### DISCUSSION TITLE
LibraVNA power Level calibration

### DISCUSSION THREAD

**Discussion by ren13ukr** [jankae/LibreVNA#220] [Q&A] [answered]
There is a power "Level" choice Up/Down box in the main toolbar of the VNA or Tracking Generator mode 
There is a power "Level" choice Up/Down box in the Generator mode
I have the R&S power meter head NRP-Z11 and Aaronia SpectranV6 to check power levels
Libra always lies with those levels, In best examples it produces ~10dB according to real measures, but usually even more
1. Is it hard to calibrate generated power levels? 
2. Does a procedure exist where I can propose to increase generator levels up to +10dBm at least?
I measure boosters sometimes (two-way amplifiers) with switch levels from -7dBm up to 9dBm and have to use a pre-amplifier to increase levels. Thank you

**Comment by jankae** [comment, marked as answer]
The output power level (and measured power level) in the LibreVNA is not calibrated by factory. For VNA measurements this usually doesn't matter too much, but you are right, in the Generator mode or Spectrum Analyzer mode this value can be off (by a lot towards the higher frequencies).
The absolute level may be wrong by several dBs, but you should always be able to change the relative output power level by adjusting the power level up/down button. If that does not change the power level at all, something is going very wrong.
If you want more precise power levels, you can calibrate them yourself, see chapter 9 in the user manual ("Amplitude calibration").
